code/modules/power/generators/turbine.dm

```diff
@@ -27,7 +27,7 @@
 #define SLOW 1
 
 //below defines the time between an overheat event and next startup
-#define OVERHEAT_TIME 60 //measured in cycles of 2 seconds
+#define OVERHEAT_TIME 1200 //measured in deciseconds (two minutes)
```

Comment:
We have a seconds define you can use here, to automatically convert it from seconds to deciseconds
```suggestion
#define OVERHEAT_TIME 120 SECONDS
```

Reply:
Fixed in lastest!